code/unclassified/src/min_max_of_an_array_using_min_comparisons/Added_comments_Maximum_and_Minimum_of_an_array_using_minimum_comparisons_(Tournament Method).cpp

```diff
@@ -0,0 +1,58 @@
+// Find maximum and minimum of an array using minimum number of comparisons
+#include <cstdlib>
+#include <iostream>
+#include <vector>
+
```

Comment:
@mehakmalhotra1 Please remove unwanted line breaks

Reply:
Please review it